scss/inc/rtl/_review-panel.scss

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2021 Open Assessment Technologies SA
+*/
+
+.runner {
+    .test-runner-scope {
+        .test-sidebar.test-sidebar-left {
+            .qti-panel.qti-navigator {
+                .qti-navigator-label {
+                    .icon-up, .icon-down {
+                        margin-left: 0;
+                        margin-right: auto;
+                    }
+                    .qti-navigator-counter {
```

Comment:
It is not necessary to have specificity for those classes 0.10.1 = 0.2.1 (_rtl.scss) + 0.8.0 (_review-panel.scss)
![Screenshot 2021-06-08 at 18 41 57](https://user-images.githubusercontent.com/2067027/121225222-98872300-c889-11eb-89c8-72dc4f008ac2.png)
I woud suggest to keep it lower as much is possible.

Or is there is a reason for this?